feature_1/hello.txt

```diff
@@ -0,0 +1 @@
+Hello feature_1 !
```

Comment:
it is really good that you have decided to add this line of code

Reply:
Thanks for your helpful comment.

Reply:
I agree with @t-tomaat  - this line provided a lot of clarity.

Reply:
It is really good that you decided to add this one